config/schema_config.yaml

```diff
@@ -191,6 +191,16 @@ super enhancer:
   is_a: non coding element
   inherit_properties: true
 
+non coding rna:
+  represented_as: node
+  input_label: non_coding_rna
+  is_a: non coding element
+  inherit_properties: true
+  properties:
+    rna_type: str
+    rfam: str[]
```

Comment:
This should be converted into a relationship

Reply:
I am not sure what relationship we can make between `ncRNA` and `rfam` since `rfam` isn't defined as entity in the schema and I don't know to which parent node to be categorized.